api-server/src/roster/models.py

```diff
@@ -387,3 +390,126 @@ class UserImage(models.Model):
 
     def __str__(self):
         return f'{self.file.name or self.url}'
+
+
+class LiveryLuaSection (models.Model):
+    name = models.CharField(max_length=128)
+    text = TextField(blank=True, null=True)
+
+    def __str__(self):
+        return self.name
+
+
+class LiverySkinJsonValidator(BaseValidator):
+    def compare(self, value, schema):
+        try:
+            jsonschema.validate(value, schema)
+        except jsonschema.exceptions.ValidationError:
+            raise ValidationError('Failed JSON schema check, please fix your JSON and try again.')
```

Comment:
the jsonschema `ValidationError` will describe the validation error that is found, we should use this to provide a hint on what needs to be fixed.

Reply:
👍